subworkflows/local/cram_sampleqc/main.nf

```diff
@@ -13,32 +12,32 @@ workflow CRAM_SAMPLEQC {
     main:
 
     versions = Channel.empty()
-    reports  = Channel.empty()
+    reports = Channel.empty()
 
     if (!skip_baserecalibration) {
 
         CRAM_QC_RECAL(
             cram,
             fasta,
-            intervals_for_preprocessing)
+            intervals_for_preprocessing,
+        )
 
         // Gather QC reports
-        reports = CRAM_QC_RECAL.out.reports.collect{ meta, report -> report }
+        reports = CRAM_QC_RECAL.out.reports.collect { _meta, report -> report }
 
         // Gather used softwares versions
         versions = versions.mix(CRAM_QC_RECAL.out.versions)
     }
 
-    BAM_NGSCHECKMATE(cram.map{meta, cram, crai -> [meta, cram]}, ngscheckmate_bed.map{bed -> [[id: "ngscheckmate"], bed]}, fasta)
-    versions = versions.mix(BAM_NGSCHECKMATE.out.versions.first())
+    BAM_NGSCHECKMATE(cram.map { meta, cram_, _crai -> [meta, cram_] }, ngscheckmate_bed.map { bed -> [[id: "ngscheckmate"], bed] }, fasta)
+    versions = versions.mix(BAM_NGSCHECKMATE.out.versions)
```

Comment:
That one led to another forgotten versions, so I removed all the `versions.first()`